gcn_monitor/kafka.py

```diff
@@ -63,7 +64,7 @@ def parse_filenames(message):
     return file_name, message_key_file_name, headers_file_name
 
 
-def run():
+def run(bucketName):
```

Comment:
Please change to snake_case to conform with Python style conventions.

```suggestion
def run(bucket_name):
```